### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by jecode93**
In this milestone we :
- Create a collection that keeps a list of books 
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Display all books saved in the collection at the top part of the page.
- When a user clicks the "Add" button:
  - A new book is added to the collection.
  - The new book is displayed on the page.
- When a user clicks the "Remove" button:
  - The correct book is removed from the collection.
  - The correct book disappears from the page.
- Make the data preserved in the browser's memory by using [localStorage]

### CODE REVIEW

File: js/add-books.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    allBooks.innerHTML += `
+    <article class="books">
+      <p>${bookStore[i].title}</p>
+      <p>${bookStore[i].author}</p>
+      <button class="remove" onclick="removeBook(${i})">Remove</button>
+      <hr>
+    </article>
+`;
+  }
+}
```
Comment (1GORDON) [inline js/add-books.js]: Great job team🤝

- [ ] When adding the second book to the collection, an extra book is rendered to the UI. Kindly fix this to make your project conform to the project requirements.

File: js/add-books.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  } else {
+    error.innerHTML = 'Please enter a book';
+  }
+}
+
+function removeBook(index) {
+  bookStore.splice(index, 1);
+  localStorage.setItem('books', JSON.stringify(bookStore));
+  showBooks();
+}
```
Comment (1GORDON) [inline js/add-books.js]: Great job team 🤝

- [ ] When the delete button is clicked to remove the book from the collection, it is removed from the local storage but they aren't removed from the UI. Kindly ensure the book removed from the local storage is also removed from the UI.
deleted

File: README.md [hunk 3]
```diff
@@ -1 +1,117 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (1GORDON) [inline README.md]: Great job team🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @jecode93, and @Bupilipili  ✋

Good job so far!

## Highlights!
- Preserved data in the local storage ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/add-books.js (hunk 1 above)
Comment (jecode93) [reply js/add-books.js]: Hello @1GORDON 

Thanks a lot for your review..

So please, How do you think we can solve this problem cause we saw it but we try to fix it but it isn't work, hope you can help us with that, please?

File: js/add-books.js (hunk 2 above)
Comment (jecode93) [reply js/add-books.js]: @1GORDON we have the same problem here, to see the changes we have to refresh the page..

How do you think we could solve this, please..?

**Review by julie-ify — APPROVED** [review]
Hi @jecode93 and @Bupilipili,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**